app/controllers/image_sets_controller.rb

```diff
@@ -1,7 +1,7 @@
 class ImageSetsController < ApiController
   before_filter :require_image_set, except: [:create, :index]
-  before_filter :require_organization, only: [:index]
-  before_filter :require_current_user_in_organization, only: [:index]
+  before_filter :require_organization, only: [:index, :destroy]
+  before_filter :require_current_user_in_organization, only: [:index, :destroy]
 
```

Comment:
@redyaffle We shouldn't use these filters here. if you look at the [code for these filters](https://github.com/201-created/LG-api/blob/master/app/controllers/api_controller.rb#L58-L67) it's expecting an organization_id in params, which is not going to be included by the front end on a delete request. 'require_image_set' sets up a variable `@image_set` so what we should validate is that `current_user.organization == @image_set.organization`.